src/order/disjoint.lean

```diff
@@ -320,6 +320,18 @@ calc a ⊓ x ≤ (b ⊔ y) ⊓ x : inf_le_inf hle le_rfl
 lemma le_sup_right_iff_inf_left_le {a b} (h : is_compl x y) : a ≤ b ⊔ y ↔ a ⊓ x ≤ b :=
 ⟨h.inf_left_le_of_le_sup_right, h.symm.dual.inf_left_le_of_le_sup_right⟩
 
+lemma sup_inf_sup_eq (h : is_compl x y) : (a ⊔ x) ⊓ (b ⊔ y) = (b ⊓ x) ⊔ (a ⊓ y) :=
```

Comment:
I think `sup_inf_sup_comm` is the standard name for this type of thing, though usually we'd order it as
```suggestion
lemma sup_inf_sup_comm (h : is_compl a d) : (a ⊔ b) ⊓ (c ⊔ d) = (a ⊓ c) ⊔ (b ⊓ d) :=
```
Maybe it's convenient to have both